src/sr25519.h

```diff
@@ -38,6 +38,11 @@ typedef struct VrfResult {
     bool is_less;
 } VrfResult;
 
+/**
+ * public_key: the public key of the keypair to sign the message, 32 bytes long
+ * private_key: the secret key of the keypair to sign the message, 64 bytes long
+ */
+void sr25519_public_key_from_private_key(sr25519_public_key public_key, sr25519_secret_key private_key);
```

Comment:
Can't understand the warning here
sr25519_public_key_from_private_key

Reply:
What warning?

Reply:
old-style parameter declarations in prototyped function definition

Reply:
I don't see that warning either in VSCode or compiling, so it's probably ok to ignore